CHANGELOG.md

```diff
@@ -36,6 +36,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 Note: You should always set `allow-deprecated-apis=false` to test for
 changes.
 
+- JSON API: `listpeers` and `listnodes` fields `localfeatures` and `globalfeatures` (now just `features`).
```

Comment:
Should we also change the plugin hooks/notifications API then ?

Reply:
(`peer_connected` for example)

Reply:
`peer_connected` was updated in 52207acd9fe6b0cb858668e081fbb92a7f77a024, the documentation is lagging however.